velox/exec/Aggregate.h

```diff
@@ -190,6 +208,17 @@ class Aggregate {
     return numNulls_ && (group[nullByte_] & nullMask_);
   }
 
+  void incrementRowSize(char* row, uint64_t bytes) {
+    VELOX_DCHECK(rowSizeOffset_);
+    uint32_t* ptr = reinterpret_cast<uint32_t*>(row + rowSizeOffset_);
+    uint64_t size = *ptr + bytes;
+    *ptr = std::min<uint64_t>(bytes, std::numeric_limits<uint32_t>::max());
```

Comment:
My reading of this method is that bytes can be larger than 32-bit integer. Why is it ok to truncate it? Also, shouldn't we update ptr to "size", not "bytes". Are there tests for this code path?